test/java/com/arpnetworking/metrics/portal/scheduling/JobExecutorActorTest.java

```diff
@@ -236,43 +236,6 @@ public void testActorRequestsStopWhenJobNoLongerExists() {
         _probe.expectTerminated(ref);
     }
 
-    @Test
-    public void testActorRequestsStopWhenJobIsDeletedBetweenRuns() {
```

Comment:
I get why you had to remove this (since this PR prevents hitting the DB) but will this still be handled by anti-entropy?

Reply:
Yes, anti-entropy should force a reload of the job as well, and so should explicit job deletion via an API. 

However, the anti-entropy runs only periodically, so a job may continue executing past its deletion time until that runs. That seems expected to me. I don't believe there are any tests for this; however, I did confirm (by code inspection) that the API for reports sends the reload and so does anti-entropy.